src/components/Contacts/ContactFilters/ContactFilters.tsx

```diff
@@ -0,0 +1,44 @@
+import { QueryLazyOptions } from '@apollo/client';
+import { colors } from '@material-ui/core';
+import React from 'react';
+import { Exact } from '../../../../graphql/types.generated';
+import { ContactFiltersQuery } from '../../../../pages/accountLists/[accountListId]/ContactFilters.generated';
+
+interface Props {
+  data: ContactFiltersQuery;
+  loading: boolean;
+  error: Error;
+  loadFilters: (
+    options?: QueryLazyOptions<Exact<{ accountListId: string }>>,
+  ) => void;
+}
+
+const ContactFilters: React.FC<Props> = ({
+  data,
+  loading,
+  error,
+  loadFilters,
+}: Props) => {
+  return (
+    <div style={{ backgroundColor: colors.amber[600] }}>
+      <h2>Filters</h2>
+      <button data-testID="LoadFiltersButton" onClick={() => loadFilters()}>
+        Load Filters
+      </button>
```

Comment:
Just adding context, I think the filters panel is collapsible and will be collapsed by default. So I was thinking we'd make it lazy load once it's open. I just added a button for now.